File: app/src/main/java/com/example/myapplication/atest/UserRegister.java
package com.example.myapplication.atest;

import com.google.gson.annotations.SerializedName;

public class UserRegister {
    public static class RequestParam {
        @SerializedName("Email")
        String email;

        @SerializedName("Password")
        String password;

        @SerializedName("ValidateCode")
        String validateCode;

        public RequestParam(String email, String password, String validateCode) {
            this.email = email;
            this.password = password;
            this.validateCode = validateCode;
        }
    }
}

File: app/src/main/java/com/example/myapplication/atestkotlin/UserRegister.kt
package com.example.myapplication.atestkotlin

import com.google.gson.annotations.SerializedName

/**
 * Copyright (c) 2022 Raysharp.cn. All rights reserved.
 *
 * UserRegister
 * @author liujianming
 * @date 2022-01-05
 */
class UserRegister {
    class RequestParam(
        @field:SerializedName("Email") var email: String, @field:SerializedName(
            "Password"
        ) var password: String, @field:SerializedName("ValidateCode") var validateCode: String
    )
}